### ISSUE TITLE
Implement CIS benchmark using falco rules

### ISSUE THREAD

**Issue by mstemm** [falcosecurity/falco#186]
Implement as much of the CIS benchmark as possible using Falco rules. Some components, specifically those that map to auditd rules, should be easy. Some will not be possible, such as checking ulimit values. But I bet we can implement a pretty significant subset.

**Comment by JPLachance** [comment]
Greetings,

I think a great addition would be to detect containers that spawn and run as root. Running as root in a container is a well known bad practice. Detecting that would help a lot.

Thanks!

**Comment by JPLachance** [comment]
Hello!

How do we un-stale it? :D

Thanks!

**Comment by mfdii** [comment]
Added the roadmap label so it won't go stale again.

**Comment by anton-abashkin-mongodb** [comment]
This would be a very welcome addition.

It is not a short task so perhaps to start out we can review the default rule set and see if any existing rules map to the CIS benchmarks? This is already done in a few places in the [k8s_audit_rules](https://github.com/falcosecurity/falco/blob/master/rules/k8s_audit_rules.yaml#L192)

Instead of (in addition to?) adding the information to the rule comments should we consider adding it as a tag? Then it would be easy to parse and generate a table for audit reports and such.

**Comment by aabashkin** [comment]
This would be a very welcome addition.

It is not a short task so perhaps to start out we can review the default rule set and see if any existing rules map to the CIS benchmarks? This is already done in a few places in the [k8s_audit_rules](https://github.com/falcosecurity/falco/blob/master/rules/k8s_audit_rules.yaml#L192)

Instead of (in addition to?) adding the information to the rule comments should we consider adding it as a tag? Then it would be easy to parse and generate a table for audit reports and such.

**Comment by daxmc99** [comment]
Circling back around on this. 
I wasn't able to find rules that are a direct equivalent to the CIS benchmarks. At least not in an organized format. The goal here would be something like Neuvector's CIS benchmarks scripts
https://github.com/neuvector/kubernetes-cis-benchmark/tree/master 

These might exist but for a noobie it hard to parse the falco rules codebase
